### DISCUSSION TITLE
can anyone do a https example?

### DISCUSSION THREAD

**Discussion by sprappcom** [vlang/v#21488] [Questions and Answers]
as titled

**Comment by JalonSolov** [comment]
Lots of examples in the `examples` subdir.

Shortest is `links_scraper.v`.  A bit more complex is `news_fetcher.v`

**Reply by sprappcom** [reply]
no i mean https server

**Reply by JalonSolov** [reply]
V doesn't do that right now.  Would involve adding a lot of code to handle certificates, etc.

The current recommendation is to use caddy or some other server as a front-end proxy to your code.

**Reply by sprappcom** [reply]
@JalonSolov would love to have vlang use in the real life practical business app. even after many years, everything seemed like for "hobby" projects

**Reply by medvednikov** [reply]
Yes, caddy or nginx. Very easy to set up, better performance.
